srcpkgs/qjackctl/template

```diff
@@ -14,6 +14,16 @@ homepage="http://qjackctl.sourceforge.net"
 distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=5196c5c01b7948c1a8ca37cd3198a7f0fe095a99a34a67086abd3466855b4abd
 
+build_options="jack_session"
+build_options_default="jack_session"
+desc_option_jack_session="Enable support for the deprecated Jack Session API (which pipewire does not implement)"
+
+if [ "$build_option_jack_session" ]; then
+	configure_args+=" --enable-jack-session"
+else
+	configure_args+=" --disable-jack-session"
+fi
+
```

Comment:
Instead of the if-else-block you should use `configure_args=(vopt_enable jack-session)`.
See [https://github.com/void-linux/void-packages/blob/master/Manual.md#build-options](https://github.com/void-linux/void-packages/blob/master/Manual.md#build-options).

Reply:
Thanks!